From 088bb4cef2c93bef312b076a8c7a3c798cbf2f19 Mon Sep 17 00:00:00 2001
From: daicy <daicy@cn.fujitsu.com>
Date: Tue, 8 Dec 2015 11:45:01 +0900
Subject: [PATCH] Makefile.am: Error Fix

Do not compile man dir since error always occur.
 | sed: file filenames.sed line 3: unterminated `s' command
 | make[2]: *** [sessreg.1] Error 1
It is not a good way.But can remove it from PNBLACKLIST.

Upstream-Status: pending

Signed-off-by: Li Xin <lixin.fnst@cn.fujitsu.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index cddffd0..94f7048 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -18,7 +18,7 @@
 #  TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
 #  PERFORMANCE OF THIS SOFTWARE.
 
-SUBDIRS = man
+#SUBDIRS = man
 bin_PROGRAMS = sessreg
 
 AM_CFLAGS = $(CWARNFLAGS) $(SESSREG_CFLAGS)
-- 
1.8.4.2

